nixos/modules/services/networking/netbird/dashboard.nix

```diff
@@ -39,8 +39,6 @@ in
 
     package = mkPackageOption pkgs "netbird-dashboard" { };
 
-    enableNginx = mkEnableOption "Nginx reverse-proxy to serve the dashboard";
```

Comment:
Should `mkRemovedOptionModule` be added for thhis?

Reply:
I've thought about it a bit more and realized that people might want to host their own dashboard, without nginx, but using the `finalDRV` so we might need the option after all.